src/rvsdg/egglog_optimizer/loop-optimizations.egg

```diff
@@ -8,7 +8,7 @@
        (let pass-through (PassThroughArguments (vec-length outputs)))
        (union theta
           (Gamma
-            (SubstOperandAll pred after-one-iter)
+            (SubstOperandAll pred (VO inputs))
```

Comment:
I agree this one is correct; just confirming the fix is intended?

Reply:
Yep, forgot to mention I fixed this!